server/main_server/src/irods_server_control_plane.cpp

```diff
@@ -20,11 +21,17 @@
 
 #include <boost/lexical_cast.hpp>
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wnarrowing"
+#include <boost/process.hpp>
+#pragma GCC diagnostic pop
+
```

Comment:
remove

create an issue about using Boost.Process to call out to external programs (e.g. *scripts/pid_age.py*)

Reply:
and consider adding that advice/best practice to the developers section of irods_docs as well.

Reply:
Will do

Reply:
done.